src/features/counter/Counter.jsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable import/prefer-default-export */

```

Comment:
Default React App w/Redux file.  Possible deletion in the future.  Linter error move export to bottom 2x